datatypes/src/primitives/coordinate.rs

```diff
@@ -2,16 +2,19 @@ use crate::util::arrow::ArrowTyped;
 use arrow::array::{BooleanArray, Float64Builder};
 use arrow::datatypes::DataType;
 use arrow::error::ArrowError;
+use ocl::OclPrm;
 use serde::{Deserialize, Serialize};
 use std::{fmt, slice};
 
-#[derive(Clone, Copy, Debug, Deserialize, PartialEq, PartialOrd, Serialize)]
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, PartialOrd, Serialize, Default)]
```

Comment:
What is a default coordinate and why should we use it?

Reply:
this is necessary to be able to copy coordinates to opencl device. the OclPrm marker trait requires data to impl Default

Reply:
Hmm, I guess it is not harmful to have this `Default` then. @jdroenner what is your opinion on that?